lib/semian/platform.rb

```diff
@@ -13,6 +13,6 @@ def semaphores_enabled?
   end
 
   def disabled?
-    ENV["SEMIAN_SEMAPHORES_DISABLED"] || ENV["SEMIAN_DISABLED"]
+    ENV.key?("SEMIAN_SEMAPHORES_DISABLED") || ENV.key?("SEMIAN_DISABLED")
```

Comment:
The PR description says that these checks were being moved to initialization. Should these env checks maybe be assigned to a constant to achieve that?

Reply:
@jgaskins I would left as it is for now. It is used in few places with few branchings, that would made PR even bigger. I am going to create a separate PR for this.

Reply:
created a separate issue for it: https://github.com/Shopify/semian/issues/442